uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -437,6 +449,16 @@ define([
     }
 
     /**
+     * Gets called when the component is destroyed
+     */
+    function unsubscribeFromEvents () {
+      // destroy all the event
```

Comment:
1) A comment to a function should describe what a function does and not from where is it called. What if we decide to call this function from another place - that would mean we must then change the comment, which does not sound right.
2) The `//` comment becomes unnecessary if we amend the function description
```js
/**
  * Destroys all event listeners
  */
```